--- a/Makefile
+++ b/Makefile
@@ -49,7 +49,7 @@
 LOADEROBJS = $(LOADEROBJ)
 
 %.o: %.c %.h
-	$(CC) -c $(CFLAGS) -fpic -o $@ $<
+	$(CC) -c $(CFLAGS) -fPIC -o $@ $<
 
 kudzu_loader.o: kudzu.c
 	$(CC) -c $(CFLAGS) -D__LOADER__ -o $@ $<
@@ -72,7 +72,7 @@
 	for ver in $(PYTHONVERS) ; do \
 		if [ ! -f "$$ver/_kudzumodule.so" -o libkudzu.a -nt "$$ver/_kudzumodule.so" ]; then \
 			mkdir -p $$ver ;\
-			$(CC) $(CFLAGS) -I/usr/include/$$ver -fpic -c -o $$ver/kudzumodule.o kudzumodule.c ;\
+			$(CC) $(CFLAGS) -I/usr/include/$$ver -fPIC -c -o $$ver/kudzumodule.o kudzumodule.c ;\
 	        	$(CC) -o $$ver/_kudzumodule.so $$ver/kudzumodule.o -shared -Wl,-soname,_kudzumodule.so -L. -lkudzu -lpci ;\
 		fi ; \
 	done
